setup.cfg

```diff
@@ -1,7 +1,7 @@
 [pycodestyle]
 [pep8]
 [flake8]
-ignore=E203,E261,E501,F405,W293,W291,W503
+ignore=E203,E261,E501,E731,F405,W293,W291,W503
```

Comment:
this allows assigning lambda's to variables

not needed anymore in this PR, but multiple people have been annoyed by that restriction, leaving it up to the reviewer to decide if we do/don't want this